fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -0,0 +1,493 @@
+                                originalSize,
+                                min(dpToPx(peekHeight), fullHeight * BottomSheetOpenFraction)
+                            )
+                        }
+                    }
+                    .sheetHeight(expandable, fullHeight, peekHeight)
+                    .semantics(mergeDescendants = true) {
+                        if (sheetState.isVisible) {
+                            dismiss {
+                                if (sheetState.confirmStateChange(BottomSheetValue.Hidden)) {
+                                    scope.launch { sheetState.hide() }
+                                }
+                                true
+                            }
+                            if (sheetState.currentValue == BottomSheetValue.Shown) {
+                                expand {
+                                    if (sheetState.confirmStateChange(BottomSheetValue.Shown)) {
+                                        scope.launch { sheetState.expand() }
+                                    }
+                                    true
+                                }
+                            } else if (sheetState.hasExpandedState) {
+                                collapse {
+                                    if (sheetState.confirmStateChange(BottomSheetValue.Expanded)) {
+                                        scope.launch { sheetState.show() }
+                                    }
+                                    true
+                                }
+                            }
+                        }
+                    },
+                shape = sheetShape,
+                elevation = sheetElevation,
+                color = sheetBackgroundColor,
+                contentColor = sheetContentColor
+            ) {
+                Column {
+                    if (showHandle) {
+                        Column(
+                            horizontalAlignment = Alignment.CenterHorizontally,
+                            modifier = Modifier
+                                .padding(vertical = 8.dp)
+                                .fillMaxWidth()
+                                .draggable(
+                                    orientation = Orientation.Vertical,
+                                    state = rememberDraggableState { delta ->
+                                        sheetState.performDrag(delta)
+                                    },
+                                    onDragStopped = { velocity ->
+                                        launch {
+                                            sheetState.performFling(velocity)
+                                            if (!sheetState.isVisible) {
+                                                scope.launch { sheetState.hide() }
+                                            }
+                                        }
+                                    },
+                                )
```

Comment:
Check if we can merge this with Drawer's fling draggable operation.

Reply:
This would need refactoring in DrawerState first. Will take it up once whole drawer module complete to optimize code reuse.

Reply:
Sure. Makes sense. Can we keep a TODO so that we remember to do this?

Reply:
Sure, will do that

Reply:
okay please add that then.

Reply:
Added TODO to revisit to abstract out common modifier.